examples/layout/src/MapView.vue

```diff
@@ -24,8 +24,7 @@ full-screen-viewport
     :expanded='panel.expanded',
     :footer='true',
     :actions='panel.actions',
```

Comment:
Hi @jbeezley, the recent PRs are really cool. Just a small thing that I think the `:actions='panel.actions'` can be removed now.

Reply:
Thanks, I'll fix remove it.